core-web/libs/block-editor/src/lib/extensions/ai-image-prompt/ai-image-prompt.component.html

```diff
@@ -4,28 +4,35 @@
         [dismissableMask]="true"
         [draggable]="false"
         [resizable]="false"
-        [style]="{ width: '800px' }"
+        [style]="{ width: '880px' }"
         (onHide)="hideDialog()"
         appendTo="body"
         header="{{ 'block-editor.extension.ai-image.dialog-title' | dm }}">
-        <div class="dialog-prompts__wrapper grid">
-            <dot-ai-image-prompt-input
-                class="col"
-                [isLoading]="vm.status === ComponentStatus.LOADING"
-                [selected]="vm.selectedPromptType === 'input'"
-                (click)="selectType('input', vm.selectedPromptType)"
-                (promptChanged)="generateImage($event)"
-                placeholder="{{ 'block-editor.extension.ai-image.input-text.placeholder' | dm }}"
-                type="input" />
-
-            <dot-ai-image-prompt-input
-                class="col"
-                [isLoading]="vm.status === ComponentStatus.LOADING"
-                [selected]="vm.selectedPromptType === 'auto'"
-                (click)="selectType('auto', vm.selectedPromptType)"
-                (promptChanged)="generateImage($event)"
-                placeholder="{{ 'block-editor.extension.ai-image.auto-text.placeholder' | dm }}"
-                type="auto" />
+        <div class="dialog-prompt__wrapper flex">
```

Comment:
```suggestion
        <div class="dialog-prompt__wrapper">
```

Add the flex in the css.